docs/topologies/star-shaped/index.md

```diff
@@ -3,56 +3,57 @@ sidebar_label: 'Star-shaped Network'
 sidebar_position: 4
 ---
 
-# Star-Shaped Head Network
+# Star-Shaped Head network
 
 :hammer_and_wrench: This document is a work in progress
 
 This document details the behaviour of so-called _Star-shaped Hydra Network_.
 
 ## Summary
 
-A _Star-shaped Hydra Network_ or more precisely a Star-shaped Heads Network is comprised of:
+A _Star-shaped Hydra Network_, or more precisely a Star-shaped Heads network, comprises:
+
+* A central _Server_ node that provides "Head-as-a-service" with low expected downtime, likely operated by a company or organization equipped to host this service.
+* _Client_ nodes, which may be DApp instances or mobile/personal wallets. These nodes might not always be online and can in
 
-* A central _Server_ node providing "Head-as-a-service", with low expected downtime, probably operated by some company or organisation with enough resources to host this service,
-* _Client_ nodes, either DApp instances, or mobile/personal wallets, which might not be always online and possibly can come and go.
 
 ![Star-shaped Heads Network](./star-shaped-general.jpg)
 
-Client nodes want to be able to interact with each other efficiently, at a low cost, using L2 solution, with all the Hydra safety guarantees, but without bearing the operational burden of operating an always online "full" Hydra node (eg. using an embedded version of the node, or a lightweight version). There might be a lot of them, say in the 100s or even 1000s but they are not always all live and up at the same time.
+Client nodes aim to interact with each other efficiently and at low cost using an L2 solution, with all the Hydra safety guarantees, but without the operational burden of maintaining an always-online "full" Hydra node. This might involve using an embedded version of the node or a lightweight variant. Potentially, hundreds or even thousands of such client nodes exist, though they are not always active simultaneously.
 
-Client nodes establish pairwise Heads (eg. _channels_) with the server: This setup is simpler than with a normal multiparty head because the server has as a well-known identity and the client can always provide the needed parameters (keys, IP) to the server when setting up the Head using some specific service whose definition is outside of the scope of this document.
+Client nodes establish pairwise Heads (e.g., _channels_) with the server. This setup is simpler than with a normal multiparty head because the server has a well-known identity, and the client can always provide the needed parameters (keys, IP) to the server when setting up the Head using a specific service, the details of which are outside the scope of this document.
```

Comment:
```suggestion
Client nodes establish pairwise heads (eg, _channels_) with the server. This setup is simpler than with a normal multiparty head because the server has a well-known identity, and the client can always provide the needed parameters (keys, IP) to the server when setting up the head using a specific service, the details of which are outside the scope of this document.
```